.zenodo.json

```diff
@@ -890,6 +890,11 @@
       "affiliation": "MIT, HMS",
       "name": "Ghosh, Satrajit",
       "orcid": "0000-0002-5312-6729"
+    },
+    {
+      "affiliation": "Department of Neurosurgery, Medical College of Wisconsin",
+      "name": "Espana, Lezlie",
+      "orcid": "0000-0002-6466-4653"
```

Comment:
I'd suggest to modify the zenodo file within its own commit. Also, I believe you should keep Satra as last author.

Reply:
Oh of course, my mistake.

Reply:
No problem